projects/igniteui-angular/src/lib/directives/date-time-editor/date-time-editor.directive.ts

```diff
@@ -500,8 +512,9 @@ export class IgxDateTimeEditorDirective extends IgxMaskDirective implements OnCh
     /** @hidden */
     protected override setPlaceholder(_value: string): void { }
 
-    private updateDefaultFormat(): void {
-        this._defaultInputFormat = DateTimeUtil.getDefaultInputFormat(this.locale);
+    private updateDefaultFormat(dataType: DataType = DataType.DateTime): void {
```

Comment:
Um wouldn't this make the editor default to date-time format if not specified, where it used to default to date-only? Don't think that'd be expected and the param isn't used, so just leave it to the old default in `getDefaultInputFormat`

Reply:
Right, I changed this, but since this is related to the default format for the `dateTime` column in the grid, thought before trying to come up with anything, it would be better to discuss how it should be addressed, after this change and according to the new implementation. Currently some tests will be failing.
So far, the input format was set to a default format property ([ref](https://github.com/IgniteUI/igniteui-angular/blob/a34e44c06d8abe05d57f3d1fa790516840ecb90e/projects/igniteui-angular/src/lib/grids/cell.component.html#L143)) defined on the FieldType interface ([ref](https://github.com/IgniteUI/igniteui-angular/blob/a34e44c06d8abe05d57f3d1fa790516840ecb90e/projects/igniteui-angular/src/lib/grids/common/grid.interface.ts#L308)). So it is hardcoded and also includes time parts.

So, if we'd like that to remain the same, I guess it should be additionally handled for the `dateTime` column?

To give some more context, the input format will not be inferred from the displayFormat in this case, as the default display format for `dateTime` col (`medium`) contains non-numeric parts (ref [here](https://github.com/IgniteUI/igniteui-angular/blob/a34e44c06d8abe05d57f3d1fa790516840ecb90e/projects/igniteui-angular/src/lib/grids/columns/column.component.ts#L62) and [here](https://github.com/IgniteUI/igniteui-angular/blob/a34e44c06d8abe05d57f3d1fa790516840ecb90e/projects/igniteui-angular/src/lib/grids/columns/column.component.ts#L1841)). So, it will resolve to the default one for the dateTimeEditorDirective, which will be date-only.

Reply:
Okay, was going to ask how the columns even manage the difference between date/date-time because either default on the editor won't work for both. That `medium` format is unfortunate, but I guess it's the only reasonable option w/ time included :/
Regardless, changing the behavior of another component to suit the grid usage probably not the way to go. Would've also been nice if the spec actually specified the default inputFormat behavior rather than a remark on the API doc. Which reminds me multiple specs are also likely due an update once we're done moving around changes.